mp4parse_capi/src/lib.rs

```diff
@@ -393,6 +393,15 @@ fn track_time_to_us(time: TrackScaledTime, scale: TrackTimeScale) -> Option<u64>
     rational_scale(time.0, scale.0, microseconds_per_second)
 }
 
+fn get_track_time_to_us(time: i64, scale: TrackTimeScale) -> i64 {
+    assert!(time >= 0);
+    let track_time = TrackScaledTime(time as u64, scale.1);
```

Comment:
I plan to use something like [1]. We may also need to make TrackScaledTime generic for both i64/u64, I think.

[1] http://stackoverflow.com/questions/36013519/how-can-i-make-a-rust-function-accept-any-floating-type-as-an-argument